charts/trino/templates/tests/test-jmx.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.jmx.exporter.enabled }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "trino.fullname" . }}-test-jmx
+  labels:
+    {{- include "trino.labels" . | nindent 4 }}
+    app.kubernetes.io/component: test
+    test: jmx
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    {{- if .Values.jmx.exporter.enabled }}
+    - name: trino-jmx
+      image: {{ include "trino.image" . }}
+      command: ["/bin/bash", "-c"]
+      args:
+      - curl -s {{ include "trino.fullname" . }}.{{ .Release.Namespace }}:5556 | grep -q trino
+    {{- end }}
+    {{- if .Values.serviceMonitor.enabled }}
+    - name: podmonitor
+      image: {{ include "trino.image" . }}
+      command: ["/bin/bash", "-c"]
+      args:
+      - sleep 60
```

Comment:
This should be a loop with a timeout. It might be better to write the whole test in Python, and run it using the `python:3-slim` image. We should not rely on the fact that Python is bundled in the Trino image.

I'm not requesting changes here, we can do that in a follow-up PR.

Reply:
Yeah this makes much more sense to me.